torch/testing/_internal/common_utils.py

```diff
@@ -4955,6 +4955,34 @@ def deco(cls):
     return deco
 
 
+def skipIfSandcastle(reason):
+    """
+    Decorator to skip the test if run in sandcastle
```

Comment:
Hi, I think it would be great to add a short description about `sandcastle` in here, make other unfamiliar with this term(like me :D) known when to use it, thanks!

Reply:
TBH I'm not fully sure what the sandcastle is. As far as I understand it is a farm of machines used for testing. Not sure if only for CI.